apps/draw-io/src/DrawIoEditor.vue

```diff
@@ -13,7 +13,7 @@
 import { mapGetters, mapActions } from 'vuex'
 import { basename } from 'path'
 import queryString from 'query-string'
-import moment from 'moment'
```

Comment:
Moment is still a dependency in Web. It can still be either used here and removed in separate PR (as it is used in e.g. files app as well) or would be nice to get rid of it already 🙂

Reply:
I would like to get rid of it already - but didn't have time anymore yesterday and wanted to see CI results. CI is failing. 😁